apps/spruce/src/components/MetadataCard.tsx

```diff
@@ -43,16 +44,23 @@ interface ItemProps {
   as?: PolymorphicAs;
   children: React.ReactNode;
   "data-cy"?: string;
+  description?: string;
 }
 
 export const MetadataItem: React.FC<ItemProps> = ({
   as = "p",
   children,
   "data-cy": dataCy,
+  description,
```

Comment:
[nit] Maybe we should call this property `tooltip` or `tooltipDescription` so its use is more clear